expressEngine/product/service/product.js

```diff
@@ -25,6 +26,26 @@ class Service {
     }
     return { data: product, message: success.productRegistered };
   }
+
+  getProduct = async ({ query }) => {
+    const dataToFind = {
+      collection: Product,
+      data: {
+        isDiscontinued: { $ne: true },
+      },
+      limit: query.limit,
+      skip: query.skip,
+    };
+    const products = (await DBService.find(dataToFind));
+    const err = { error: error.noRecords };
```

Comment:
@aman-successive Please place err object in if condition using it.

Reply:
updated it